src/Razor/src/Microsoft.VisualStudio.Mac.LanguageServices.Razor/VisualStudioMacEditorDocumentManager.cs

```diff
@@ -26,8 +27,8 @@ protected override ITextBuffer GetTextBufferForOpenDocument(string filePath)
                 throw new ArgumentNullException(nameof(filePath));
             }
 
-            // VS for Mac doesn't currently support find all references support yet.
-            return null;
+            var document = IdeApp.Workbench.GetDocument(filePath);
```

Comment:
Interesting... Why does this fix the issue though? I imagine this is just a workaround to a more in-depth issue in MonoDevelop right?